commit 14d2df5ec1a67ed54e8e84f735b382654bed93a2
Author: Alex Riesen <alexander.riesen@cetitec.com>
Date:   Thu Dec 21 09:29:39 2017 +0100

    Add Raspberry Pi firmware driver to the dependencies of backlight driver
    
    Otherwise the backlight driver fails to build if the firmware
    loading driver is not in the kernel
    
    Signed-off-by: Alex Riesen <alexander.riesen@cetitec.com>

diff --git a/drivers/video/backlight/Kconfig b/drivers/video/backlight/Kconfig
index e345a32945b7..9244f2f7b642 100644
--- a/drivers/video/backlight/Kconfig
+++ b/drivers/video/backlight/Kconfig
@@ -267,6 +267,7 @@ config BACKLIGHT_PWM
 
 config BACKLIGHT_RPI
 	tristate "Raspberry Pi display firmware driven backlight"
+	depends on RASPBERRYPI_FIRMWARE
 	help
 	  If you have the Raspberry Pi DSI touchscreen display, say Y to
 	  enable the mailbox-controlled backlight driver.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1158_14d2df5ec1a67ed54e8e84f735b382654bed93a2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1158_14d2df5ec1a67ed54e8e84f735b382654bed93a2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
